preflight_checks.sh

```diff
@@ -58,6 +69,11 @@ if [ "$machine" == "linux" ]; then
     fi
 fi
 
+check_port 3000
+check_port 5432
+check_port 6379
```

Comment:
Do `5432` and `6379` need to be available on the host? We don't expose these ports (note that this does differ between our `dev` and `main` branch -- on `dev` they are exposed).

Reply:
Good question, I was wondering that myself. I suppose we don't need to if it's something customers won't run into.

Reply:
I _think_ we won't need them, but I'm not a docker network expert so I do think it's worth testing. 

If ports only exposed internally are exposed then no, there shouldn't be any others. If all ports used within the platform need be available on the host, then there are a few: `8110`, `8100`, and I think both `strelka-coordinator` and `strelka-frontend` would have a port.

Reply:
Yeah, I think this should only be affected by forwarded ports.

Reply:
Got it down to just 3000 and 8000.